features/step_definitions/appointment_summary_steps.rb

```diff
@@ -0,0 +1,91 @@
+When(/^I visit the digital appointment summary generator$/) do
+  @page = Pages::AppointmentSummaryGenerator.new
+  @page.load
+end
+
+When(/^I select (.*) as my age range$/) do |age|
+  generator = @page.generator
+
+  case age
+  when '55 or over' then generator.appointment_type_standard.set true
+  when '50 to 54' then generator.appointment_type_50_54.set true
+  end
+end
+
+When(/^I view the appointment summary$/) do
+  generator = @page.generator
+  generator.submit_button.click
+
+  @page = Pages::AppointmentSummary.new
+end
+
+Given(/^I generate a valid appointment summary$/) do
+  step 'I visit the digital appointment summary generator'
```

Comment:
As a rule it's usually best to avoid nested steps at the cost of duplication. You could potentially either use `background`, duplicate those steps or just delegate them to actual methods instead. Breaking this things up later on becomes real tricky when steps call steps call steps.